install/relay-credentials.sh

```diff
@@ -13,11 +14,12 @@ if [[ ! -f "$RELAY_CREDENTIALS_JSON" ]]; then
   # JSON. We hit this case as we redirect output to the same config folder,
   # creating an empty credentials file before relay runs.
 
+  docker run --rm --volume "$(pwd)/$RELAY_DIRECTORY:/tmp/relay" busybox chown -R 10001 /tmp/relay
```

Comment:
Why not put these in a custom `entrypoint` script and use `$dcr` instead?

Reply:
Does this (https://github.com/getsentry/self-hosted/pull/1267/commits/30ff93adb28bb62562e2854c4e398b8323c29b04) look good?